app/controllers/administrator/categories_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Administrator::CategoriesController < ApplicationController
+  
+  def new
+    @category = Category.new
+  end
+
+  def create
+    @category = Category.new(category_params)
+    if @category.save
+      flash[:success] = "Successfully Create Category."
+      redirect_to administrator_categories_path
```

Comment:
Please use 'administrator_categories_url' instead of 'administrator_categories_path'